tests/runner.py

```diff
@@ -923,24 +923,25 @@ def _test_dylink_dso_needed(self, do_run):
     # reset dylink-related options first.
     self.clear_setting('MAIN_MODULE')
     self.clear_setting('SIDE_MODULE')
-    self.clear_setting('RUNTIME_LINKED_LIBS')
 
     # XXX in wasm each lib load currently takes 5MB; default INITIAL_MEMORY=16MB is thus not enough
     self.set_setting('INITIAL_MEMORY', '32mb')
 
     so = '.wasm' if self.is_wasm() else '.js'
 
     def ccshared(src, linkto=[]):
-      cmdv = [EMCC, src, '-o', shared.unsuffixed(src) + so] + self.get_emcc_args()
-      cmdv += ['-s', 'SIDE_MODULE', '-s', 'RUNTIME_LINKED_LIBS=' + str(linkto)]
+      cmdv = [EMCC, src, '-o', shared.unsuffixed(src) + so, '-s', 'SIDE_MODULE'] + self.get_emcc_args()
+      cmdv += linkto
       self.run_process(cmdv)
 
     ccshared('liba.cpp')
     ccshared('libb.cpp', ['liba' + so])
     ccshared('libc.cpp', ['liba' + so])
 
     self.set_setting('MAIN_MODULE')
-    self.set_setting('RUNTIME_LINKED_LIBS', ['libb' + so, 'libc' + so])
+    original_args = list(self.emcc_args)
```

Comment:
```suggestion
    original_args = self.emcc_args[:]
```

Reply:
I find the former more clear and pythonic myself.   I've not seen the later outside the emscripten codebase.. but I will do a little research.  I could be wrong.

Reply:
Looks like both methods are fairly commonly used.. but python3 has a `list.copy()` method which is probably better than both of these .. I'll followup with a PR to use that across the board.